tests/FVTTests/constants.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+export const USERNAME :string = process.env.ZOWE_USERNAME;
+export const PASSWORD :string = process.env.ZOWE_PASSWORD;
+
+export const BASE_URL :string = `https://${process.env.SERVER_HOST_NAME}:${process.env.SERVER_HTTPS_PORT}`;
```

Comment:
can be used value from above USERNAME PASSWORD, instead of reaching out to process.env again

Reply:
instead of constants.ts can we call this file config.ts  or environment.ts

Reply:
sorry, i was reading wrong SERVER_HOST_NAME, SERVER_HTTPS_PORT as username password!